torch/csrc/jit/codegen/cuda/type.cpp

```diff
@@ -64,6 +64,16 @@ static const char* val_type2string(ValType t) {
       return "Scalar";
     case ValType::NamedScalar:
       return "NamedScalar";
+    case ValType::KirIterDomain:
```

Comment:
is this really needed?

Reply:
Yes, for ir_iostream, an IR that isn't printable isn't debug-able.